hub/templates/hub/area.html

```diff
@@ -94,174 +94,61 @@ <h5 class="card-title mb-3">MP profiles</h5>
                     </div>
                 </div>
             </div>
-            <div class="col d-flex flex-column">
-                <a href="#" class="bg-transparent rounded p-3 flex-grow-1 d-flex align-items-center justify-content-center" style="border: 2px dashed var(--bs-border-color-translucent);">
-                    <span>…3 more datasets available</span>
-                </a>
-            </div>
         </div>
 
+      {% if categories.opinion or counts.opinion %}
         <h2 class="mt-5 mb-3 text-primary">Public opinion</h2>
-
         <div class="row row-cols-1 row-cols-md-2 row-cols-lg-3 g-4">
+          {% for category in categories.opinion %}
+            {% include 'hub/area/_place_data.html' with category=category %}
```

Comment:
Oh, one thing – there’s nothing "place" specific in this file. Do you think this should just be `_data.html` or `_datapoint.html` or something, to avoid future confusion?

Reply:
I've updated this in a later branch to be `_opinion_data.html` as I'm assuming that they are all going to have their own templates